### DISCUSSION TITLE
requirements.txt에 fake-useragent 버전을 수정해야 할 것으로 보입니다.

### DISCUSSION THREAD

**Discussion by ISeeTheFuture** [josw123/dart-fss#170] [General]
0.4.9 릴리즈하시면서 useragent 초기화 시에 platforms 인자 주신 것 때문에, 1.4 버전까지는 못씁니다.

정확한 버전 확인은 안해봤는데, fake-useragent 최신버전 1.51 버전을 써야 오류가 안났던 것으로 기억합니다.

**Comment by josw123** [comment]
안녕하세요.

최소 버전확인후 곧바로 반영하도록 하겠습니다!

감사합니다!
